src/rantanplan/pipeline.py

```diff
@@ -29,21 +29,25 @@ def custom_tokenizer(nlp):
 _load_pipeline = {}
 
 
-def load_pipeline(lang=None):
+def load_pipeline(lang=None, use_affixes=True):
     """
     Loads the new pipeline with the custom tokenizer
     :param lang: Spacy language model
+    :param use_affixes: Wether or not use spacy_affixes
     :return: New custom language model
     """
     global _load_pipeline
     if lang is None:
         lang = 'es_core_news_md'
     if lang not in _load_pipeline:
         nlp = spacy.load(lang)
-        nlp.tokenizer = custom_tokenizer(nlp)
-        nlp.remove_pipe("affixes") if nlp.has_pipe("affixes") else None
-        suffixes = {k: v for k, v in load_affixes().items() if k.startswith(AFFIXES_SUFFIX)}
-        affixes_matcher = AffixesMatcher(nlp, split_on=["VERB"], rules=suffixes)
-        nlp.add_pipe(affixes_matcher, name="affixes", first=True)
+        if use_affixes:
+            nlp.tokenizer = custom_tokenizer(nlp)
+            nlp.remove_pipe("affixes") if nlp.has_pipe("affixes") else None
+            suffixes = {k: v for k, v in load_affixes().items() if
+                        k.startswith(AFFIXES_SUFFIX)}
+            affixes_matcher = AffixesMatcher(nlp, split_on=["VERB"],
+                                             rules=suffixes)
```

Comment:
The line `nlp.tokenizer = custom_tokenizer(nlp)` should remain out of the `if` block I believe.